pkg/metrics/tags.go

```diff
@@ -43,3 +46,27 @@ var (
 	PodNameKey       = tag.MustNewKey(metricskey.PodName)
 	ContainerNameKey = tag.MustNewKey(metricskey.ContainerName)
 )
+
+var (
+	allowedEventTypes = []string{
```

Comment:
How about a map[string]struct{} instead? Then you just need to lookup membership in the map, rather than iterating.

Reply:
Yeah, it seemed weird to me that there was no slice contains() function, this seems better

Reply:
k8s has a set in "k8s.io/apimachinery/pkg/util/sets"
no need to, just fyi

Reply:
I prefer sets.String{} too, but some don't like adding the extra dependency.